backend/src/test-integration/java/com/crm/api/v1/CustomerControllerIntegrationTest.java

```diff
@@ -0,0 +1,35 @@
+package com.crm.api.v1;
+
+import com.crm.App;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.security.test.context.support.WithMockUser;
+import org.springframework.test.web.servlet.MockMvc;
+
+import static org.springframework.http.HttpStatus.NOT_FOUND;
+import static org.springframework.http.HttpStatus.OK;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@SpringBootTest(classes = App.class)
+@AutoConfigureMockMvc
+@WithMockUser
+public class CustomerControllerIntegrationTest {
+
+    @Autowired
+    private MockMvc mvc;
+
+    @Test
+    public void shouldReturnCorrectResponseStatusWhenCallingCustomerWithExistingId() throws Exception {
+        mvc.perform(get("/customers/1"))
```

Comment:
Maybe I missed something, but is the response empty? Maybe we can consider smth like this ->
.andExpect(jsonPath("$.pageable.id", is(1)))

Reply:
Thanks. I will check it later

Reply:
The endpoint customers/{id} does not return a pageable object. Could you explain?

Reply:
Done